File: python/sivitasakademik.py
from human import Human


class SivitasAkademik(Human):
    # constructor sivitasAkademik
    def __init__(self, nik, name, jenis_kelamin, asal_univ, email):
        # pemanggilan variabel kelas parent
        super().__init__(nik, name, jenis_kelamin)
        self.asal_univ = asal_univ
        self.email = email

    # getter/setter methods untuk class Sivitas akademik
    def get_asal_univ(self):
        return self.asal_univ

    def get_email(self):
        return self.email

File: CPP/sivitasakademik.cpp
#include "header.hh"

// constructor for SivitasAkademik class
SivitasAkademik::SivitasAkademik(string nik, string name, string jenis_kelamin, string asal_univ, string email)
    : Human(nik, name, jenis_kelamin)
{
    this->asal_univ = asal_univ;
    this->email = email;
}

// getter methods for SivitasAkademik class
string SivitasAkademik::get_asal_univ()
{
    return this->asal_univ;
}

string SivitasAkademik::get_email()
{
    return this->email;
}